.github/workflows/ci.yml

```diff
@@ -87,3 +87,33 @@ jobs:
       - run: cargo run -- --help
       - run: cargo run -- list-remote node
       - run: cargo run -- list-remote wasm-test
+  build:
+    name: Upload build
+    runs-on: ${{ matrix.host }}
+    strategy:
+      matrix:
+        include:
+        - target: aarch64-unknown-linux-gnu
+          host: ubuntu-latest
+        - target: x86_64-unknown-linux-gnu
+          host: ubuntu-latest
+        - target: x86_64-apple-darwin
+          host: macos-latest
+        - target: aarch64-apple-darwin
```

Comment:
does this actually work? i dont think it does by default without extra configuration.

Reply:
D'oh! I left off the `--target` from `cargo install`